lib/matplotlib/axes/_axes.py

```diff
@@ -876,6 +876,55 @@ def _to_points(xy1, xy2, slope):
         self._request_autoscale_view()
         return line
 
+    def axline_transaxes(self, xy1, slope, **kwargs):
+        """
+        Add an infinitely long straight line.
+
+        The line is defined by one point in axes coordinates *xy1* and a
```

Comment:
I think I understand the matplotlib transformation stack relatively well, but the term "axes coordinates"  was too quickly scanned over when I read this, and I think it will be completely mysterious to a casual user.  So maybe:

```
The line is defined by a *slope* in data coordinates, set by the current x- and y-limits of the axes, and a point in axes-relative coordinates, which is in the same place on the axes regardless of x- and y-limits.  
```

Reply:
added in 527aa460d25dfbd43229bdae6308b162db9eb902.